baseframe/static/js/baseframe-material.js

```diff
@@ -58,7 +58,7 @@ function activate_widgets() {
   });
 
   // Activate codemirror on all textareas with class='stylesheet'
-  $('textarea.stylesheet').each(function() {
+  $('textarea.stylesheet:not([style*="display: none"])').each(function() {
```

Comment:
What is the reason for this exception to Codemirror? What broke? The `:not` selector is expensive, so maybe we should explicitly add a `codemirror` class or a `data-widget="codemirror"` attribute and look for it.

Reply:
For dynamic sub label forms in funnel, code mirror  has to be initialised again but that was causing already changed textarea to be initialised again and two code mirror instances were created. `:not` selector was to avoid initialising already initialised textarea.

Reply:
We have removed description field from label model, so I will revert this change.